Install.rst: Install virtualenv with --user

lilithxxx opened the pull request:
Updates documentation to tell user to Install virtualenv with --user and how to check whether its already installed.

Closes coala/coala#4017

<!--
Thanks for your contribution!

Please take a quick look at those things down there. They're quite important.
Really! We wrote them for you. Yes you! With utmost care. Read them.
-->

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

Commit:
Install.rst: Install virtualenv with --user

Updates documentation to tell user to Install virtualenv with --user and how to check whether its already installed.

Closes https://github.com/coala/coala/issues/4017

newbazz reviewed (commented):
You havent followed the commit guidelines ;)

lilithxxx commented:
@newbazz What needs to be resolved?

newbazz reviewed (commented):
Read the commit guidelines and follow them http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

kriti21 commented on Users/Install.rst:
```diff
@@ -90,13 +90,21 @@ Virtualenv is probably what you want to use during development.
+::
+
+    $ pip3 install --user virtualenv
+
```
Only one blank line is needed here. Please delete the extra empty line.

pareksha commented on Users/Install.rst:
```diff
@@ -90,13 +90,20 @@ Virtualenv is probably what you want to use during development.
+
+If you see a version number, its already installed. If you do
+not, this can be done with pip3 easily:
+
```
You can improve this by mentioning the output that the user will see if ``virtualenv`` is not installed.

lilithxxx replied:
I have already mentioned that the user will not see a version number if `virtualenv` is not installed and I think the output message would be pretty obvious to know that its not installed yet

newbazz commented on Users/Install.rst:
```diff
@@ -90,13 +90,20 @@ Virtualenv is probably what you want to use during development.
+    $ virtualenv --version
+
+If you see a version number, its already installed. If you do
+not, this can be done with pip3 easily:
```
You need to explain the use of `--user` flag.

Commit:
Install.rst: Install virtualenv with --user 

Updates documentation to tell user to Install virtualenv with --user and how to check whether its already installed.

Closes https://github.com/coala/coala/issues/4017

jayvdb reviewed (changes requested):
pls squash your two commits into a single commit.

jayvdb commented:
@gitmate-bot rebase

jayvdb commented on Users/Install.rst:
````diff
@@ -132,13 +132,22 @@ steps given below.
+
+    $ pip3 install --user virtualenv
+
+The ```--user``` tag is required to install it with root access.
````
with -> without

jayvdb reviewed (changes requested):
One small fix, and please squash your commits, and rebase.
